source/slang/glsl.meta.slang

```diff
@@ -2821,6 +2821,453 @@ public vec4 shadow2DProjLod(sampler2DShadow sampler, vec4 coord, float lod)
     return textureProjLod(sampler, coord, lod);
 }
 
+//// gimageDim Implementation
+
+public const int GL_TEXTURE_CUBE_MAP_POSITIVE_X = 0;
```

Comment:
`public static const int`.

Due to the legacy of HLSL design, global `const`s are treated as uniform parameters. `static const` makes them actual constant enums.